tslint.json

```diff
@@ -21,7 +21,7 @@
         ],
         "quotemark": [
             true,
-            "double"
+            "single"
```

Comment:
I prefer double, it's more standard, see this very json file. Also seems more natural coming from C#.
Some strong reason to change?